fwd

```diff
@@ -135,6 +135,21 @@ elif [ "$1" == "dump" ]; then
             mysql \
             bash -c 'MYSQL_PWD=$MYSQL_ROOT_PASSWORD mysqldump -u root --default-character-set=utf8mb4 $MYSQL_DATABASE'
     fi
+    
+# If "restore" is used, run mysql
+# from our mysql container
+# It expects the dump file path as the second arg
+# e.g.: ./fwd restore /path/to/dump-file.ext
+elif [ "$1" == "restore" ]; then
+    shift 1
+    if [ "$IS_RUNNING" == "yes" ]; then
+        cat $1 | $COMPOSE_EXEC -T \
```

Comment:
I don't see that as a problem. It's just two ways of restoring a dump.

1 - When you want to kee yous local changes AND
2 - When you want to fully restore a backup;

I would say that when you want to fully restore a backup, the `drop database` statement is usually at the beginning of the dump script file.

The dump file could be written in so many ways. I think it's up to the developer to know what's in the dump he/she is running.

Another suggestion would be having a second arg indicating what type of restoring should be performed.

I've used the `-T` flag because I was running the `exec` command. It was not working without it. The `<` was not working either. I could do it like the `dump` command and use `exec` when the container is running and `run` when it's not.

Reply:
The dumps never get a drop database on top of it, mainly because of permission issue on the Mysql user's. (notice the way I suggested based on a project with that).

The added/removed table issue is a thing I encountered couple of times already in the main scenario that comes to mind - getting a dump from prod or staging and restoring it locally. Due to the nature of Laravel and the migrations table, I stand for the suggestion of by default drop / create the database - although I guess there could be a case where you just want to restore it without dropping the whole database.

Given this, what about we create two commands? The `restore` one as suggested here IMHO needs to indeed restore our local DB as of the dump, precisely, that is, dropping the whole database to make sure that happens.

We could then have a `import` or something for that effect, that is the current suggested simply importing a dump on top of the current existing database.

What do you think?

Reply:
Makes sense. I'll implement that.

Reply:
Done!